packages/shared/src/utils/login/login.js

```diff
@@ -4,9 +4,11 @@ import { isMobileOs } from '../os/os_detect';
 import { getAppId, domain_app_ids } from '../config/config';
 import { getStaticUrl, urlForCurrentDomain } from '../url';
 
-export const redirectToLogin = (is_logged_in, language) => {
+export const redirectToLogin = (is_logged_in, language, has_params = true) => {
     if (!is_logged_in && isStorageSupported(sessionStorage)) {
-        sessionStorage.setItem('redirect_url', window.location.href);
+        const l = window.location;
+        const redirect_url = has_params ? window.location.href : `${l.protocol}//${l.host}${l.pathname}`;
```

Comment:
I think both `window.location.href` and `${l.protocol}//${l.host}${l.pathname}` are same

Reply:
No they aren’t the same. In my case I need to remove query params from the url. For example if the url is this: 
`example.com/page?param1=some_value`
And we use `window.location.href` the result will be `example.com/page?param1=some_value`
But if we use `${l.protocol}//${l.host}${l.pathname}` the result is this `example.com/page`